.lintrunner.toml

```diff
@@ -0,0 +1,251 @@
+[[linter]]
+code = 'FLAKE8'
+include_patterns = ['**/*.py']
+exclude_patterns = [
+    '.git/**',
+    '**/*.pyi',
```

Comment:
Didn't we only include pattern  `*.py`, why are we excluding `*.pyi`?

Reply:
🤷🏼 pytorch does.

Reply:
hahaha, we don't have to blindly follow them